### PR TITLE
(FM-7605) - Disabling conf_enabled on Ubuntu 18.04  by default as it conflicts with Shibboleth causing errors with apache2.

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

**Comment by david22swan** [comment]
screen shot 2018-12-06 at 12 43 36 pm

File: manifests/params.pp [hunk 1]
```diff
@@ -218,7 +218,11 @@
     $server_root         = '/etc/apache2'
     $conf_dir            = $httpd_dir
     $confd_dir           = "${httpd_dir}/conf.d"
-    $conf_enabled        = "${httpd_dir}/conf-enabled"
+    if !($::operatingsystem == 'Ubuntu' and $::lsbdistrelease == '18.04') {
```
Comment (HelenCampbell) [inline manifests/params.pp]: This should be moved to the Ubuntu18 specific params section further down.

File: manifests/params.pp [hunk 2]
```diff
@@ -218,7 +218,11 @@
...
+      $conf_enabled      = "${httpd_dir}/conf-enabled"
+    } else {
+      $conf_enabled      = ''
```
Comment (eimlav) [inline manifests/params.pp]: Could this line not go in the block for Ubuntu 18 at line 300, and then the original $conf_enabled value left untouched?

**Comment by david22swan** [comment]
@HelenCampbell @eimlav Code is update

**Comment by david22swan** [comment]
Have removed the exclusion to it previous location as the new one caused unit test failures.

**Comment by david22swan** [comment]
@ekohl Are you happy with the code as it is now??

**Pull request merged**